nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -156,7 +162,7 @@
                     + "FlowFile. The 'Use statement.type Attribute' option is the only one that allows the 'SQL' statement type. If 'SQL' is specified, the value of the field specified by the "
                     + "'Field Containing SQL' property is expected to be a valid SQL statement on the target database, and will be executed as-is.")
```

Comment:
We should add some doc here explaining that some Statement Type values may not be supported depending on the value of the Database Type property, and can even include an example ("For example, UPSERT statements are not supported by Oracle" or something like that)

Reply:
Also probably some text saying "please refer to the database documentation for a description of the behavior of each operation." This is because even the different DBs that `do` support "upsert" may do them differently, for instance PostgreSQL requires a unique or exclusion constraint on the update key column(s)

Reply:
I'll add the general info to the description but rather not add the example because this part of the code has no direct connection to the adapters so I think it should not comment on _their_ functionality. (The Oracle adapter - or any other that we might refer in the example - might get updated to support UPSERT and almost certainly we won't remember to update this doc.)